.gitattributes

```diff
@@ -0,0 +1,21 @@
+# Package manager lock files
+Pipfile.lock linguist-generated
+package-lock.json linguist-generated
+
+# Generated GraphQL schemas
+*.graphql linguist-generated
```

Comment:
I'm 👎 on this particular change. That's the only way to understand how a backend change affects the API.